entry.sh

```diff
@@ -39,6 +39,11 @@ export LD_LIBRARY_PATH=$LDMX_SW_INSTALL/lib:$LD_LIBRARY_PATH
 export PYTHONPATH=$LDMX_SW_INSTALL/python:$LDMX_SW_INSTALL/lib:$PYTHONPATH
 export PATH=$LDMX_SW_INSTALL/bin:$PATH
 
+#add what we need for GENIE 
+export GENIE=/usr/local/GENIE/Generator
```

Comment:
The environment variable `GENIE` should already be defined from the `ENV GENIE ...` line in the Dockerfile.

```suggestion
```

Reply:
This has been removed, btw.